pipeline/scripts/create_postprocessing_dataframes.jl

```diff
@@ -1,8 +1,11 @@
-using EpiAwarePipeline, EpiAware, JLD2, DrWatson, DataFramesMeta, CSV
+using EpiAwarePipeline, EpiAware, JLD2, DrWatson, DataFramesMeta, CSV, MCMCChains
 
 ## Define scenarios
 scenarios = ["measures_outbreak", "smooth_outbreak", "smooth_endemic", "rough_endemic"]
 
+## Define true GI means
+true_gi_means = [2.0, 10.0, 20.0]
```

Comment:
This isn't connect to this PR but I see this and the scenarios being specified in a few repeated places now?

Reply:
The goal is that this script runs the other postprocessing scipts via `include` and as I hit repetitions in either the script or underlying functions I'm moving into single calls to avoid DRY. e.g. in this PR I found myself reusing some code and created the `_get_info_from_config` internal function to do this.

So anywhere you spot repetitions are in scope to bug me about (I'm going to look for a code analysis tool to try and keep this tidy).

Reply:
In this case, I've rewritten the top script to be based on the actual pipeline types we use which seems much more robust